examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
```

Comment:
<p>"rounded to integer" doesn't really make sense. Instead the procedure to get an actual cut is to measure in the computational basis, which splits into two subsets depending on whether whether the qubit is measured to be 0 or 1; the negative loss is the average expected cut size.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>